src/main/java/com/vinylteam/vinyl/dao/jdbc/JdbcUserDao.java

```diff
@@ -16,9 +16,11 @@
 
 public class JdbcUserDao implements UserDao {
 
-    private static final Logger logger = LoggerFactory.getLogger(JdbcUserDao.class);
-    private static final UserRowMapper userRowMapper = new UserRowMapper();
-    private static final String FIND_BY_EMAIL = "SELECT email, password, salt, iterations, role, status, discogs_user_name" +
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    private final UserRowMapper userRowMapper = new UserRowMapper();
+    private final String COUNT_ALL = "SELECT COUNT(*) FROM public.users";
```

Comment:
Why remove static, and where we use COUNT_ALL?

Reply:
seems to me it's a merge issue, thanks